src/main/scala/org/scalajs/tools/tsimporter/Trees.scala

```diff
@@ -181,7 +181,7 @@ object Trees {
 
   case class ConstructorMember(signature: FunSignature) extends MemberTree
 
-  case class IndexMember(indexName: Ident, indexType: TypeTree, valueType: TypeTree) extends MemberTree
+  case class IndexMember(indexName: Ident, indexType: TypeTree, valueType: TypeTree, readonly: Boolean) extends MemberTree
```

Comment:
I would prefer to use a `modifiers: Modifiers`, like for the other members.

Reply:
I agress for future expansion (`private`, `static` or something)